### PR TITLE
Bump eslint to 8.22

### PR DESCRIPTION
**Pull request by Divailo**
## Overview

Outdated eslint version is a blocker for us to update the version on our side

## Changes

- Bump eslint to `8.22.0`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   "devDependencies": {
     "@skyscanner/eslint-config-skyscanner": "^11.1.0",
     "babel-eslint": "^10.0.1",
-    "eslint": "^7.32.0",
+    "eslint": "^8.22.0",
```
Comment (jaysonwu991) [inline package.json:29]:
```suggestion
    "eslint": "8",
```
Test CI running.

File: package.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
   "devDependencies": {
     "@skyscanner/eslint-config-skyscanner": "^11.1.0",
     "babel-eslint": "^10.0.1",
-    "eslint": "^7.32.0",
+    "eslint": "8",
```
Comment (jaysonwu991) [inline package.json:29]:
```suggestion
    "eslint": "^8.22.0",
```

File: package-lock.json [hunk 3]
```diff
@@ -1244,6 +1193,17 @@
         "node": ">=10.10.0"
       }
     },
+    "node_modules/@humanwhocodes/gitignore-to-minimatch": {
+      "version": "1.0.2",
+      "resolved": "https://artifactory.skyscannertools.net/artifactory/api/npm/npm/@humanwhocodes/gitignore-to-minimatch/-/gitignore-to-minimatch-1.0.2.tgz",
```
Comment (jaysonwu991) [inline package-lock.json:1198]: You have to switch the official npm registry because of open-source.

Comment (Divailo) [reply package-lock.json:1198]: Good catch

**Pull request closed without merging**